xml/obs_best_practice_kiwi_editor.xml

```diff
@@ -9,45 +9,165 @@
    <mediaobject>
     <imageobject>
-     <imagedata fileref="edit_kiwi_package_overview.png"/>
+     <imagedata fileref="kiwi_package_overview.png"/>
     </imageobject>
    </mediaobject>
   </figure>
   <note>
    <para> You should see now a "Edit KIWI" link (next to "Delete package"
     link). </para>
   </note>
-  <para> Click on the "Edit KIWI" link. </para>
+  <para> Click on the "Edit KIWI" link and you will be redirected to the Editor.</para>
   <figure>
    <title>&kiwi; Editor. Show screen</title>
    <mediaobject>
     <imageobject>
-     <imagedata fileref="edit_kiwi_show.png"/>
+     <imagedata fileref="kiwi_editor_show.png"/>
     </imageobject>
    </mediaobject>
   </figure>
-  <para> Click on the "Edit" button. </para>
+  <para> We can see two different sections:
+    <itemizedlist>
+     <listitem>
+      <para>
+        <emphasis>Repositories:</emphasis> Display the repositories set in the Kiwi file.
+      </para>
+      </listitem>
+     <listitem>
+      <para>
+        <emphasis>Packages:</emphasis> Display the packages of the package group with type <emphasis>image</emphasis>.
+      </para>
+     </listitem>
+    </itemizedlist>
+  </para>
+  </sect2>
+  <sect2>
+  <title>Adding Repositories in the &kiwi; Editor</title>
+  <para> To add a new repository you only need click to <emphasis>Add repository</emphasis> link and fill the dialog.
+    There are two ways to create it:
+    <itemizedlist>
+      <listitem>
+       <para>
+          <emphasis>Basic Mode:</emphasis> Adding the name of a project will provide a list with the repositories from that project.
+       </para>
+      </listitem>
+    </itemizedlist>
+  </para>
   <figure>
-   <title>&kiwi; Editor. Edit screen</title>
+   <title>&kiwi; Adding a new repository - Basic Mode</title>
    <mediaobject>
     <imageobject>
-     <imagedata fileref="edit_kiwi_edit.png"/>
+     <imagedata fileref="kiwi_editor_repo_basic.png"/>
     </imageobject>
    </mediaobject>
   </figure>
-  <para> Modify the &kiwi; configuration as you want, and click the "Save" button
-   when you finish. </para>
- </sect2>
+  <itemizedlist>
+    <listitem>
+      <para>
+        <emphasis>Expert Mode:</emphasis> This mode provides you with a set of customizable parameters for creating a repository.
+      </para>
+      <itemizedlist>
+        <listitem>
+         <para>
+           <emphasis>Type:</emphasis> Valid options are <emphasis>rpm-md</emphasis> and <emphasis>apt-deb</emphasis>.
+         </para>
+        </listitem>
+        <listitem>
+         <para>
+            <emphasis>Priority:</emphasis> Repository priority for the given repository.
+         </para>
+        </listitem>
+        <listitem>
+         <para>
+            <emphasis>Alias:</emphasis> Alternative name for the configured repository.
+         </para>
+        </listitem>
+        <listitem>
+         <para>
+            <emphasis>Source Path:</emphasis> Define the repository path
```

Comment:
Do you need a period at the end of this sentance? I don't mind either way but all the other points here end with periods..